.circleci/publish_helm_charts.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+REVISION=${GITHUB_TAG:-$GIT_SHA1}
+if [ -n ${GITHUB_TAG} ]; then
+  COMMIT_MESSAGE="Updating release to $REVISION"
+else 
+  COMMIT_MESSAGE="Updating release to $REVISION"
```

Comment:
I guess these two messages are supposed to be different?

Reply:
Oops yeah - good catch